tests/appsec/iast/test_iast.py

```diff
@@ -45,7 +45,7 @@ def test_insecure_hash_multiple(self):
             r, vulnerability_count=2, type="WEAK_HASH", location_path=self.EXPECTED_LOCATION
         )
 
-    @missing_feature(library="nodejs", reason="Need to be implement global vulnerability deduplication")
+    @missing_feature(context.library < "nodejs@3.3.1", reason="Need to be implement global vulnerability deduplication")
```

Comment:
@uurien paternity leave
@CarlesDD PTO until 27th
I think the commit that fix this test is merged in master: https://github.com/DataDog/dd-trace-js/commit/a721792b0eebfcf1a42422730b7b331e75f26618